ecs-logs-collector.sh

```diff
@@ -489,7 +489,7 @@ enable_ecs_agent_debug()
   case "${os_name}" in
     amazon)
 
-      if [ -e /etc/ecs/ecs.config ] &&  grep -q "ECS_LOGLEVEL=debug" /etc/ecs/ecs.config
+      if [ -e /etc/ecs/ecs.config ] &&  grep -q "^ECS_LOGLEVEL=debug" /etc/ecs/ecs.config
```

Comment:
Similar to above, lets match on `^\s*ECS_LOGLEVEL=debug` to catch people with indenting.